bundles/CoreBundle/EventListener/ResponseHeaderListener.php

```diff
@@ -57,4 +60,27 @@ public function onKernelResponse(ResponseEvent $event)
             $response->headers->set($header->getKey(), $header->getValues(), $header->getReplace());
         }
     }
+
+    public function onKernelControllerArguments(ControllerArgumentsEvent $event): void
+    {
+        $request = $event->getRequest();
+        /** @var $configurations ResponseHeader */
+        if (!$attributes = $request->attributes->get('_response_header')) {
+            return;
+        }
+
+        foreach ($attributes as $attribute) {
+            if (!$attributes instanceof ResponseHeader) {
```

Comment:
```suggestion
            if (!$attribute instanceof ResponseHeader) {
```
likely a typo, but actually in  both usage (annotation/attribute) is always instanceof `ResponseHeader`